packages/components/src/confirm/component.tsx

```diff
@@ -0,0 +1,80 @@
+// @ts-nocheck (for now)
+
+/**
+ * External dependencies
+ */
+// eslint-disable-next-line no-restricted-imports
+import { Ref, useEffect } from 'react';
+import { confirmable } from 'react-confirm';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../button';
+import type { OwnProps } from './types';
+import { Card, CardHeader, CardFooter } from '../card';
+import { Heading } from '../heading';
+import { contextConnect, PolymorphicComponentProps } from '../ui/context';
+import { useConfirm } from './hook';
+import type { KeyboardEvent } from 'react';
+
+// @todo add type declarations for the react-confirm functions
+function Confirm(
+	props: PolymorphicComponentProps< OwnProps, 'div', false >,
+	forwardedRef: Ref< any >
+) {
+	const {
+		role,
+		wrapperClassName,
+		overlayClassName,
+		show,
+		proceed,
+		confirmation,
```

Comment:
A better name for this prop would be `message`.

Reply:
👍🏻